server/views/serviceProviderReferrals/templates/serviceProviderSessionsTemplate.njk

```diff
@@ -0,0 +1,68 @@
+{% from "govuk/components/skip-link/macro.njk" import govukSkipLink -%}
```

Comment:
Is this `serviceProviderSessionsTemplate.njk` template a modified copy of an existing one? If so, it would be super-useful to mention where it's copied from in the commit message.